domains/saxophone.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "saxophones are the best instrument",
+  "owner": {
+    "username": "saxophone-guy",
+    "email": "itsarchit@keemail.me"
+  },
+  "record": {
+    "CNAME": "saxophone-guy.github.io",
+    "MX": [
+      "mx1.forwardemail.net",
+      "mx2.forwardemail.net"
+    ],
+    "TXT": "forward-email=itsarchit@keemail.me"
```

Comment:
You cannot use CNAME with any other records

Reply:
Done!

Reply:
You didn't do anything though?

Reply:
Sorry about that! I edited my changes locally but forgot to actually push. Please check again.